MapboxVision/src/main/java/com/mapbox/vision/view/VisionView.kt

```diff
@@ -103,7 +103,7 @@ constructor(
         }
 
         updateBitmap(rgbaBytes, imageSize)
-        handler.post {
+        handler?.post {
```

Comment:
This is a crashfix, right?

Reply:
I got a crash that wasn't in crash-request, but it might(and was) a cause of crash

Reply:
Would you like add `?` to all handler usages in VisionView then?

Reply:
refactored `handler.post{} -> post{}`